.env

```
@@ -19,3 +19,7 @@ VITE_APP_COLLAB_PATH=/api/private/ws/socket.io
 
 VITE_APP_COLLAB_DOC_URL=http://localhost:3000
 VITE_APP_COLLAB_DOC_PATH=/api/private/hocuspocus
+
+# AI assistant (004-web-ai-assistant) — ships OFF; PROD rollout flips it ON
+VITE_APP_ASSISTANT_ENABLED=false
+VITE_APP_ASSISTANT_BASE_PATH=/api/private/rest/assistant
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_APP_ASSISTANT_BASE_PATH key should go before the VITE_APP_ASSISTANT_ENABLED key

(UnorderedKey)